scripts/globals/spells/trust/shantotto.lua

```diff
@@ -1,16 +1,26 @@
 -----------------------------------------
 -- Spell: Shantotto
 -----------------------------------------
--- require("scripts/globals/trust");
+	require("scripts/globals/trusts");
 -----------------------------------------
 
 function onMagicCastingCheck(caster,target,spell)
-    return 0
+    if (not caster:canUseMisc(dsp.zoneMisc.PET)) then
+        return dsp.msg.basic.CANT_BE_USED_IN_AREA
+    elseif (not caster:hasTrust(dsp.trust.SHANTOTTO)) then
+		local msg = caster:checkTrust(dsp.trust.SHANTOTTO);
+		if(msg > 0) then
+			return msg
+		end
+    else
+        return 299
+    end
+	return 0;
 end
 
 function onSpellCast(caster,target,spell)
     -- Todo: put Trusts's spawn in dialog here or in entity script?
     -- Todo: automate entity spawn in core for trust spells?
-    caster:spawnTrust(75) -- see pet_list.sql, this should be 896 but the pet list won't let me do that
+    caster:spawnTrust(dsp.trust.SHANTOTTO) -- see pet_list.sql, this should be 896 but the pet list won't let me do that
```

Comment:
update/remove comments when the info they provided is no longer valid

Reply:
Fixed